### PR TITLE
docs: convert from issue template to issue form

### PR DESCRIPTION
**Pull request by Panquesito7**
## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes

- Convert from issue template to issue form.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/add-an-awesome-github-profile.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: "Add an Awesome GitHub Profile"
+description: "Add your awesome GitHub profile to the project"
+title: "[PROFILE ADDITION] <name here>"
+labels: []
```
Comment (schmelto) [inline .github/ISSUE_TEMPLATE/add-an-awesome-github-profile.yml:4]: we could think of adding a profile add label here

**Comment by schmelto** [comment]
Also do not forget to update the link in the readme

**Comment by schmelto** [comment]
Also do not forget to update the link in the readme

File: .github/ISSUE_TEMPLATE/add-an-awesome-github-profile.yml (hunk 1 above)
Comment (Panquesito7) [reply .github/ISSUE_TEMPLATE/add-an-awesome-github-profile.yml:4]: Yeah. That's a good idea. :)
Should we name it `profile addition`, `new profile`, or `new awesome profile`?

**Comment by Panquesito7** [comment]
> Also do not forget to update the link in the readme

Which link? Is it the badge link?

File: .github/ISSUE_TEMPLATE/add-an-awesome-github-profile.yml (hunk 1 above)
Comment (schmelto) [reply .github/ISSUE_TEMPLATE/add-an-awesome-github-profile.yml:4]: I think profile addition is a good name

**Comment by schmelto** [comment]
> > Also do not forget to update the link in the readme
> 
> 
> 
> Which link? Is it the badge link?
> 
> 

Oh yes sorry for the confusion

**Comment by Panquesito7** [comment]
> > > Also do not forget to update the link in the readme
> > 
> > 
> > Which link? Is it the badge link?
> 
> Oh yes sorry for the confusion

It's fine :) I'll be making a good first issue for that.

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**